src/test/java/com/exasol/adapter/dynamodb/DynamodbAdapterTestAwsIT.java

```diff
@@ -57,10 +57,29 @@ static void beforeAll() throws SQLException, BucketAccessException, InterruptedE
     @Test
     void testCountAllRows() throws SQLException {
         final ResultSet resultSet = exasolTestInterface.getStatement()
-                .executeQuery("SELECT COUNT(\"KEY\") FROM OPENLIBRARY");
+                .executeQuery("SELECT \"KEY\", \"REVISION\" FROM OPENLIBRARY");
+        resultSet.next();
```

Comment:
What does it test?

Reply:
This are performance tests. I only use them to measure the runtime